starters/lightweight-service/src/app.rs

```diff
@@ -3,6 +3,7 @@ use loco_rs::{
     app::{AppContext, Hooks},
     boot::{create_app, BootResult, StartMode},
     controller::AppRoutes,
+    environment::Environment,
```

Comment:
Revert all your changes in the state template. 

The starter template should always be aligned with the released version and not include breaking changes.

for more information, you can start [readme](https://github.com/loco-rs/loco/blob/master/starters/README.md)

Reply:
Do you mind if I the following pattern for the starter templates?

* Put a TODO before all "breaking changes", this will let other developers know that if they are trying to work with the bleeding edge changes, they will be able to see what to change, and that we haven't forgotten an item
* Go ahead and add any imports that will be needed in the next release, if the change is backwards compatible